src/shared/components/entry/EntryTypeIcon.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC } from 'react';
+import { SwissProtIcon, TremblIcon, UniParcIcon } from 'franklin-sites';
+
+import {
+  EntryType,
+  getEntryTypeFromString,
+} from '../../../uniprotkb/adapters/uniProtkbConverter';
+
+import './styles/entry-type-icon.scss';
+
+export const EntryTypeIcon: FC<{
+  entryType?: EntryType | string;
+  title?: string;
+}> = ({ entryType, title }) => {
+  let entryTypeToCheck = entryType;
+  if (typeof entryType === 'string') {
+    entryTypeToCheck = getEntryTypeFromString(entryType);
+  }
+
+  if (entryTypeToCheck === EntryType.REVIEWED) {
+    return (
+      <span
+        className="entry-title__status icon--reviewed"
+        title={title || 'This marks a reviewed entry'}
+      >
+        <SwissProtIcon />
+      </span>
+    );
+  }
+  if (entryTypeToCheck === EntryType.UNREVIEWED) {
+    return (
+      <span
+        className="entry-title__status icon--unreviewed"
+        title={title || 'This marks an unreviewed entry'}
+      >
+        <TremblIcon />
+      </span>
+    );
+  }
+  if (entryTypeToCheck === EntryType.UNIPARC) {
+    return (
+      <span
+        className="entry-title__status icon--unreviewed"
```

Comment:
`icon--unreviewed` is wrong here. It either requires a `icon--uniparc` class added in Franklin, or colouring to be applied from the uniprot-website codebase

Reply:
I'll add an `icon--uniparc`, I think it's more consistent.

Reply:
Added to franklin, will be available next time we publish